test/tic_tac_toe/io_test.clj

```diff
@@ -0,0 +1,24 @@
+(ns tic-tac-toe.io-test
+  (:require [clojure.test :refer :all]
+            [tic-tac-toe.io :refer :all]))
+
+(def message "message")
+(def clear-message (format "\033[2J"))
+
+(deftest handles-output-to-user
+  (testing "receives and handles output to user"
+    (is (= message (with-out-str (output message))))))
+
+(deftest handles-input-from-user
+  (testing "receives and handles input from user"
+    (with-in-str message
```

Comment:
I haven't used `with-in-str`, so I am not as familiar with it, but from a quick look, I think we will want to apply the same rule to `with-in-str` that we will apply to `with-redefs`

Reply:
I got rid of the io tests because I found that I would be testing the mock io.  The [UI tests](https://github.com/saralein/tic-tac-toe_clojure/pull/1/files#diff-c76be95098989bdafb535fcf2a85dc18R1) no longer use `with-in-str` and `with-out-str`.